src/vs/workbench/contrib/chat/browser/chatWidget.ts

```diff
@@ -693,6 +697,18 @@ export class ChatWidget extends Disposable implements IChatWidget {
 		}).filter(isDefined);
 
 		this._register(this.chatWidgetService.register(this));
+		this._register(this.chatWidgetService.onDidBackgroundSession(e => {
```

Comment:
If ChatWIdget is going to drive this, then we probably don't want that event because it's just driven by the ChatWidget firing its own event anyway.